lib/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@
 ##-*****************************************************************************
 
 # Include the source dirs to make the library root dir an include
-INCLUDE_DIRECTORIES(${ALEMBIC_LIBS_SOURCE_DIR})
+INCLUDE_DIRECTORIES(${ALEMBIC_LIBS_SOURCE_DIR} ${PROJECT_BINARY_DIR}/lib)
```

Comment:
I got this error building AbcOpenGL, which includes <Alembic/Util/All.h>

alembic/lib/Alembic/Util/Foundation.h:39:33: error: Alembic/Util/Config.h: No such file or directory